<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="GameWorld" name="gameWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>590</height>
     </rect>
    </property>
    <widget class="QPushButton" name="restartGameButton">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>10</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>S</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backToMenuButton">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>10</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
    <widget class="QLabel" name="answer">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>80</y>
       <width>120</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>32</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="submitButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>150</y>
       <width>411</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit Answer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addTermButton">
     <property name="geometry">
      <rect>
       <x>1100</x>
       <y>10</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Box</string>
     </property>
    </widget>
    <widget class="QWidget" name="endWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1200</width>
       <height>800</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:grey;</string>
     </property>
     <widget class="QPushButton" name="startButton_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>220</y>
        <width>180</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>RESTART</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backToMenuButton_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>310</y>
        <width>180</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>MENU</string>
      </property>
     </widget>
     <widget class="QPushButton" name="quitButton_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>400</y>
        <width>180</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>QUIT</string>
      </property>
     </widget>
     <widget class="QLabel" name="youWinText">
      <property name="geometry">
       <rect>
        <x>475</x>
        <y>120</y>
        <width>250</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>YOU WIN!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="currentLevel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string> Level 1/12</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="menuWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <widget class="QLabel" name="Gamename">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>300</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>32</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Atlas Math</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>220</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>START</string>
     </property>
    </widget>
    <widget class="QPushButton" name="helpButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>310</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>HELP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quitButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>400</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>QUIT</string>
     </property>
    </widget>
    <widget class="QWidget" name="helpWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>561</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:grey;</string>
     </property>
     <widget class="QPushButton" name="backToMenuFromHelp">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>470</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="helpMenu">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>541</width>
        <height>371</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;HELP MENU&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;======================= KeyBoard =======================&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;1. Right / Left / Up Arrow : Character Movement&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;2. Q Keyboard Button : Pick Box&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;======================= Button =======================&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;1. S : Restart game button(immediately)&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;2. B : Go back to the start menu&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;3. Add Box : Add equation variable in the box to the middle button.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;4. Answer : The button below the answer you should make.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;When you think you make equation correct, &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;press this button, then check if it is correct or not.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GameWorld</class>
   <extends>QWidget</extends>
   <header>gameworld.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
